lib/kwtool_web/templates/upload/index.html.eex

```diff
@@ -0,0 +1,23 @@
+<h2>Upload</h2>
+
+<div class="row">
+  <div class="col-sm">
+    <%= form_for @conn, Routes.upload_path(@conn, :create), [multipart: true], fn f -> %>
+    <div class="form-group">
+      <label>Keywords file with extension csv</label>
```

Comment:
Maybe we can use helper here.

```eex
<%= label f, :keyword_file, "Keywords file with extension csv" %>
```

The benefit is the for attributes is auto-generated properly 👍